src/big/big_decimal.cr

```diff
@@ -15,8 +15,10 @@ end
 # The general idea and some of the arithmetic algorithms were adapted from
 # the MIT/APACHE-licensed [bigdecimal-rs](https://github.com/akubera/bigdecimal-rs).
 struct BigDecimal < Number
-  ZERO                       = BigInt.new(0)
-  TEN                        = BigInt.new(10)
+  private TWO  = BigInt.new(2)
+  private FIVE = BigInt.new(5)
+  private TEN  = BigInt.new(10)
```

Comment:
Removing these constants from the public API is abreaking change. If we want to make them private, they can only be deprecated for now.
I'm not convinced we should hide them, though.

Reply:
Deprecation it is, then. I don't think anyone really expects `BigDecimal`'s numeric constants not to be `BigDecimal`s though.

Reply:
Yeah, that seems certainly correct 😆

Reply:
This part is actually resolved by #11820